tests/test_generic/tests/test_xml.py

```diff
@@ -470,3 +471,41 @@ def _check_dates_are_relative(self, root, file_name):
                         model_name,
                         file_name,
                     )
+
+    def _check_static_values_in_inputs(self, root, file_name):
+        allowed_inputs = {"resourceCapacity"}
```

Comment:
Minor edit, feel free to ignore.
```suggestion
        ALLOWED_INPUTS = ["resourceCapacity"]
```
Use caps case to set variable to const in python.
If you have a small number of elements using list declaration instead of set is better the practice because it has less overhead.